packit_service/worker/events/event.py

```diff
@@ -359,7 +360,7 @@ def get_package_config(self) -> Optional[PackageConfig]:
             project=self.project,
             reference=self.commit_sha,
             pr_id=self.pr_id,
-            fail_when_missing=False,
+            fail_when_missing=self.fail_when_config_file_missing,
             spec_file_path=spec_path,
         )
```

Comment:
What about adding an optional argument to this method instead of introducing a new attribute?

(Just an idea to make this more readable...)